package.json

```diff
@@ -39,13 +39,13 @@
     "@formatjs/intl-pluralrules": "^5.0.1",
     "@formatjs/intl-relativetimeformat": "^11.0.3",
     "@fortawesome/ember-fontawesome": "^0.4.1",
-    "@fortawesome/free-brands-svg-icons": "^6.2.0",
-    "@fortawesome/free-regular-svg-icons": "^6.2.0",
-    "@fortawesome/free-solid-svg-icons": "^6.2.0",
-    "@fortawesome/pro-duotone-svg-icons": "^6.2.0",
-    "@fortawesome/pro-light-svg-icons": "^6.2.0",
-    "@fortawesome/pro-regular-svg-icons": "^6.2.0",
-    "@fortawesome/pro-solid-svg-icons": "^6.2.0",
+    "@fortawesome/free-brands-svg-icons": "6.2.0",
```

Comment:
i assume pinning those versions is intentional.

Reply:
Nope. I have no idea why that happened. Ugh. I'll look into it.